### DISCUSSION TITLE
[@types/node] has incorrect type for diagnostics_channel.ChannelListener

### DISCUSSION THREAD

**Discussion by bengl** [DefinitelyTyped/DefinitelyTyped#59005] [Issues with a @types package] [answered]
The message and name params should be swapped, as per the docs.

https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/node/diagnostics_channel.d.ts#L60

https://nodejs.org/dist/latest-v17.x/docs/api/diagnostics_channel.html#channelsubscribeonmessage

[a comment by an automated account is not shown]

**Comment by stefee** [comment, marked as answer]
Fixed by https://github.com/DefinitelyTyped/DefinitelyTyped/pull/59796
